src/schemas/transaction.yaml

```diff
@@ -1,3 +1,95 @@
+Transaction7702Signed:
```

Comment:
There is no `Transaction7702Unsigned`, this should be added

Reply:
@jochem-brouwer Do you mean it will be Unsigned or need to add another Transaction7702Unsigned ?

Reply:
See the other transaction types, they have an unsigned and a signed variant :)